### ISSUE TITLE
Consumer present in @TestConfiguration reported as BeanProducerMustBeDeclaredInConfigurationComponent

### ISSUE THREAD

**Issue by pmihalcin** [buschmais/jqa-spring-plugin#23]
Given I have test configuration in my spock test class

```
@TestConfiguration
static class SpockDetachedMockConfig {
	def mockFactory = new DetachedMockFactory()

	@Bean
	Consumer<CacheUpdate> cacheUpdateNotificator() {
		return mockFactory.Mock(Consumer)
	}
}	
```

it is reported as
```
[ERROR] Constraint: spring-injection:BeanProducerMustBeDeclaredInConfigurationComponent
[ERROR] Severity: MAJOR
[ERROR] Bean producer methods must be declared in configuration components.
```

**Comment by DirkMahler** [comment]
Has been fixed as part of #24 which is part of the already release 1.5.0-RC-1, i.e. test code is no longer considered by the constraint.

**Comment by pmihalcin** [comment]
@DirkMahler 
I've just tried new version `1.5.0-RC-1` and I can confirm that `BeanProducerMustBeDeclaredInConfigurationComponent` is no longer reported - it works as expected.
This issue can be closed.
Thanks for the fix!
